### ISSUE TITLE
Find and fix any hand-coded JSON by string concatenation

### ISSUE THREAD

**Issue by haraldnagel** [MCLD/greatreadingadventure#31]
This is in Avatar.cs:

```c#
var s = "{" + string.Format("text: \"{0}\",value: {1},selected: {2},imageSrc: \"/images/Avatars/sm_{3}.png\""
    , ds.Tables[0].Rows[i]["Name"].ToString()
    , ds.Tables[0].Rows[i]["AID"].ToString()
    , selectedStr
    , ds.Tables[0].Rows[i]["AID"].ToString()
    )+ "}";
JSON = (JSON.Length == 0 ? s : JSON + "," + s);
```

We should be using a JSON library to write JSON.

**Comment by haraldnagel** [comment]
The specific JSON referenced in this issue was fixed with 65cc31b but the rest of the project should be checked for any other `string.Format`-written JSON.

**Comment by haraldnagel** [comment]
I have not run across any additional JSON in the project so I'm closing this issue. If any future JSON-by-string-concatenation is found please reopen or create a new issue.
